### ISSUE TITLE
About updates.

### ISSUE THREAD

**Issue by huixiancheng** [charlesChen02/RandLA-pytorch#1]
Nice work！ 
Would you like to point out that **Related Updates** about **”fix some bugs in the original implementation“**?

**Comment by huixiancheng** [comment]
It is great to see more Pytorch code implementations in related fields.
Also, I recently made a personal implementation of two other backbone networks, [BAF-LAC](https://github.com/Xiangxu-0103/BAF-LAC) and [BAAF](https://github.com/ShiQiu0419/BAAF-Net), and since I'm not sure the details are fully aligned, would you be willing to review the associated PR?

**Comment by charlesChen02** [comment]
Hi huixian, 

Thank you for your comment!
1. There are mainly minor bugs related to the PyTorch version. As the original repo did not indicate which specific PyTorch version to use, I just picked PyTorch 1.12 for this repo, and in this version, an error would be thrown in [here](https://github.com/charlesChen02/RandLA-pytorch/blob/58329cfa3d8dbf1a3c44676901d0cbe84dbc5b91/utils/data_process.py#L180) where the weight was expected to be in shape [19] instead of [19,1]. 
I also modified the architecture of the LFA model a lite based on this [#issue](https://github.com/QingyongHu/RandLA-Net/issues/19#issue-581693079). 
Also, the saving strategy was a bit glitchy at least when I was trying to run it. [here](https://github.com/tsunghan-wu/RandLA-Net-pytorch/blob/75adeacdb796db07e69ba990c36409c5d3ee886b/train_SemanticKITTI.py#L135) the idea is to save the highest mIoU model as the checkpoint in `self.highest_miou`, but in my practice, this variable was not updated correctly, so it keeps saving the latest model regardless of its performance. I modified to [this](https://github.com/charlesChen02/RandLA-pytorch/blob/58329cfa3d8dbf1a3c44676901d0cbe84dbc5b91/train_SemanticKITTI.py#L186) solves the problem. 
2. I am still a novice in this field and this repo is still ongoing, but I am happy to have a look at them if you don't  mind:smile:
